packages/cli/README.md

```diff
@@ -28,20 +28,21 @@ npx @qiwi/nexus-cli <arguments>
 @qiwi/nexus-cli --config=some/path/config.json --package.repo=npm --package.name=bat --package.group=quz --package.range='<2.0.3'
 ```
 ### Options
-| Option                                      | Description                            |
-|---------------------------------------------|----------------------------------------|
-| `nexus.username`, `nexus.password`          | Nexus API credentials                  |
-| `nexus.url`                                 | Nexus API URL                          |
+| Option                                      | Description                                      |
+|---------------------------------------------|--------------------------------------------------|
+| `nexus.username`, `nexus.password`          | Nexus API credentials                            |
+| `nexus.url`                                 | Nexus API URL                                    |
 | `nexus.rateLimit`                           | Components API `deleteComponent` method multiple call limit. If exists, limitation will be applied. See more at [push-it-to-the-limit](https://github.com/antongolub/push-it-to-the-limit). |
-| `package.repo`                              | name of package repository             |
-| `package.name`                              | package name                           |
-| `package.group`                             | package group                          |
-| `package.range`                             | package versions range to be deleted   |
-| `config`                                    | path to config file                    |
-| `yes`                                       | process without asking questions       |
+| `package.repo`                              | name of package repository                       |
+| `package.name`                              | package name                                     |
+| `package.group`                             | package group. If you want to get packages, not belonging to any group, set it to `null`                                    |
+| `package.range`                             | package versions range to be deleted             |
+| `config`                                    | path to config file                              |
+| `no-prompt`                                 | disable destructive action confirmation (delete) |
 
-All options are mandatory until `config` option presents.
+All options, expect `no-prompt` and `config`, are mandatory until `config` option presents.
```

Comment:
All options except `--no-prompt` must be set through the CLI flags or `--config` JSON data.